src/tests/universal-nrp-e2e/StartMSFTTests.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# StartMSFTTests.sh
```

Comment:
>StartMSFTTests.sh [](http://example.com/codeflow?start=2&length=17)

"StartInternalTests.sh"?

Avoid 'MSFT' in names

Reply:
I just left it as `StartTests.sh`